### ISSUE TITLE
Problem with saving pdf on the same location as nw.js .exe

### ISSUE THREAD

**Issue by jraack** [nwjs/nw.js#5772]
Hello guys,

I have a weird problem with my nw.js packaged app.
I make use of the library pdfmake to create pdf files. When i try to save the pdf in the exact same location i get the following error message; (debug.log)

```
[0309/150831:ERROR:registration_protocol_win.cc(53)] CreateFile: The system cannot find the file specified. (0x2)
[0314/111613:ERROR:registration_protocol_win.cc(53)] CreateFile: The system cannot find the file specified. (0x2)
```

When i try to save it to another location on my hard drive then it saves the pdf correctly.
Any suggestions?

I use the NW.js v0.14.4 for supporting XP

Best, Joe

**Comment by jraack** [comment]
Hi,
I just found out that Desktop is pointing to the root directory of the app.
So i can save everywhere but on Desktop.

Best, Joe

**Comment by Christywl** [comment]
@jraack , could you please provide a sample to reproduce this issue?

**Comment by jraack** [comment]
@Christywl I have tested the app as a standalone exe build with http://enigmaprotector.com/ 
It looks like it has to do something with this standalone version.

When i run it from the directory with all (nw) source files it works. Looks like enigmaprotector is pointing the desktop reference to the root directory of my app. Very strange.
I have also tested a version build with electron, and this version is working a standalone exe.
